tracer/src/Datadog.Trace/Debugger/Caching/LFUEvictionPolicy.cs

```diff
@@ -0,0 +1,142 @@
+// <copyright file="LFUEvictionPolicy.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+
+namespace Datadog.Trace.Debugger.Caching
+{
+    internal class LFUEvictionPolicy<TKey> : IEvictionPolicy<TKey>
```

Comment:
styling nit, but the .NET naming convention is `LfuEvictionPolicy` 😅

Reply:
Thanks! Fixed in 93ce326e53379d4935cf896c25e5600ea89b4164